src/Command/SiteImportCommandController.php

```diff
@@ -42,8 +42,19 @@ public function fromFileCommand($file)
                     $this->outputLine('Emptied database table "' . $config['table'] . '"');
                 }
                 foreach ($config['entries'] as $entry) {
-                    $conn->insert($config['table'], $entry);
-                    $this->outputLine('Added ' . json_encode($entry) . ' to database table ' . $config['table']);
+                    if ($mode === 'update' && isset($entry['uid'])) {
```

Comment:
Could we output something if "mode" is update but the uid is not set? Then people know it's a misconfiguration.

Reply:
Done!